oemer/ete.py

```diff
@@ -4,6 +4,10 @@
 import urllib.request
 from pathlib import Path
 from PIL import Image
+from numpy import ndarray
```

Comment:
`numpy` is a third party library, and should be in the second group.

Reply:
PIL also seems to be 3rd party so I moved that one as well